src/main/webapp/js/user-page-loader.js

```diff
@@ -70,6 +70,12 @@ function fetchMessages() {
 
 /** Fetches data and populates the UI of the page. */
 function buildUI() {
+<<<<<<< HEAD
+  addLinks(); 
+=======
+  addFeedLinktoNavigation();
+  addLoginOrLogoutLinkToNavigation();
+>>>>>>> 2029e34cefdd1c5b2a55008bd06b634a60c8f3c2
```

Comment:
Looks like there's a merge conflict in this file now--you can check the CodeU site for more details, but basically you have the long lines of merge markers (<<<, ===, >>>) around the old and new versions of the code. You need to edit this so only the correct code and not the <<<, ===, >>> markers show up, then run your local server to check everything is still working.